force-app/main/default/reports/EDRDReports.reportFolder-meta.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Thank you @deepakmulamalla for the effort you are gonig to on this project to separate the reports and views! @ckleinacn